### PR TITLE
fix: remove references to unused binaries appsody, kn, kubectl

### LINKED ISSUE #69: Remove appsody from source
We are no longer using appsody - so all of this code should probably be removed from the repo

### PR DESCRIPTION
**Pull request by lkingland**
Also exports the buildpack runtimes map as `Runtimes` in order to be used in the shell completion utilities.  And creates a stubbed Docker runner in place of that provided by appsody.  (Resolves #69)

### CODE REVIEW

File: docker/runner.go [hunk 1]
```diff
@@ -7,36 +7,36 @@ import (
...
 	}
 
-	// Extra arguments to appsody
+	// Extra arguments to docker
 	args := []string{"run"}
 
 	// If verbosity is enabled, pass along as an environment variable to the function.
 	if n.Verbose {
-		args = append(args, []string{"--docker-options", "-e VERBOSE=true"}...)
+		args = append(args, []string{"-e VERBOSE=true"}...)
```
Comment (lance) [inline docker/runner.go:35]: Should this also have `-rm -it -p 8080:8080`?

**Comment by lance** [comment]
@lkingland ping - looks like this needs to be rebased.

**Comment by lkingland** [comment]
Thanks Lance, I'll be reading and merging this (and the other PR in the same situation) shortly. @lance

File: docker/runner.go (hunk 1 above)
Comment (lkingland) [reply docker/runner.go:41]: Yes.  I believe the interactive flag -i won't be allowed, but the others yes.  This has been updated in the latest version.

**Pull request merged**